### ISSUE TITLE
Please add novelfulldotnet back into plugin

### ISSUE THREAD

**Issue by hypergreatthing** [LNReader/lnreader-plugins#1215]
### Describe your suggested feature

Please add old source back into the plugin for 2.0 located here:
https://github.com/LNReader/lnreader/blob/v1.1.19/src/sources/en/novelfulldotnet.ts

### Other details

strangely enough novelfull.com is different than novelfull.net.  Search results differ and therefore content isn't the same.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/LNReader/lnreader/issues/new/choose).
- [X] I have updated the app to version **[1.1.19](https://github.com/LNReader/lnreader/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by error7404** [comment]
there is already novelfull.com are sites different?

**Comment by hypergreatthing** [comment]
Yes.  Feel free to verify yourself by searching for the keyword "dimensional".  There's about 2x the results on .net
